### ISSUE TITLE
Do not use QUERY_ALL_PACKAGES permission

### ISSUE THREAD

**Issue by fatherOfLegends** [Manveer1999/flutter_sms_mms#2]
Currently this plugin is adding this permission to the manifest:
`<uses-permission android:name="android.permission.QUERY_ALL_PACKAGES" />`

This is a restricted permission when targeting Android API >= 30. Google Play now requires targeting API 33+
Please follow the guide here to specify the correct limited permissions so consumers do not need to hack away QUERY_ALL_PACKAGES: 
https://developer.android.com/training/package-visibility

**Comment by UmairSaqibBhutta** [comment]
Same issue.
Playstore rejected my app due to this permission.
Google clearly mentions that  **Less broad app-visibility method should be used**.

**Comment by fatherOfLegends** [comment]
@UmairSaqibBhutta just so you know it is possible to work around this issue by specifying a tag to remove this permission when merging manifests. I still think it should be fixed though.

**Comment by UmairSaqibBhutta** [comment]
I also tried to remove permission by using this in manifest.
    `<uses-permission android:name="android.permission.QUERY_ALL_PACKAGES" tools:node="remove" />`
but it's not working.

**Comment by rbarzel** [comment]
@UmairSaqibBhutta `tools:node="remove"` worked for me. Did you remember to also change the manifest tag?

```
<manifest xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools">
```

**Comment by Manveer1999** [comment]
This issue has been fixed so, closing this Issue in `sms_mms: 0.0.3`
